src/Converters/Converter.php

```diff
@@ -10,7 +10,7 @@ class Converter extends \League\CommonMark\Converter {
 	 *
 	 * @param string $commonMark
 	 *
-	 * @return string
+	 * @return \GrahamCampbell\Markdown\Responses\MarkdownString
```

Comment:
Unfortunately, we can't just change this type, because it now violates the interface.

Reply:
This is really a limitation of PHP. :/

Reply:
What we really want here is an instance of `Stringish`, like in hacklang.